src/utils/processDescription/emptyDescription/test.js

```diff
@@ -1,4 +1,3 @@
-// @flow
```

Comment:
It's the file that we are currently migrating, so instead of fixing flow errors I just disabled it as it

Reply:
Ah... also it's a test. I'm no migrating tests to TS, because after the component gets migrated it will be useless to check flow typing, as all of those were replaced by typescript types.